pkg/manifests/manifests.go

```diff
@@ -0,0 +1,168 @@
+package manifests
+
+import (
+	"bytes"
+	"io"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+
+	tunedv1alpha1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1alpha1"
+)
+
+const (
+	TunedNamespace          = "assets/tuned/01-namespace.yaml"
+	TunedServiceAccount     = "assets/tuned/02-service-account.yaml"
+//	TunedSecurityContextConstraints = "assets/tuned/03-scc-tuned.yaml"
```

Comment:
is this commented out on purpose?

Reply:
Yes.  At this point, I'm not sure if the operator needs or even should do this.  However, the privileged scc is needed to ensure proper functionality.  Thanks for looking!